servers/jax-rs-tests/src/test/java/org/projectnessie/jaxrs/tests/AbstractTestDatabaseAdapterRest.java

```diff
@@ -24,7 +24,7 @@
 
 /** See {@link AbstractTestRest} for details about and reason for the inheritance model. */
 @ExtendWith(DatabaseAdapterExtension.class)
-abstract class AbstractTestJerseyRest extends AbstractRestSecurityContext {
+abstract class AbstractTestDatabaseAdapterRest extends AbstractRestSecurityContext {
```

Comment:
imo both old and new name read a bit odd in that `Test` is somewhere in the middle... `Test` prefix or suffix would be more obvious